frontend/src/components/MonacoEditor/MonacoEditor.vue

```diff
@@ -23,13 +23,8 @@ import {
   useNamespacedState,
 } from "vuex-composition-helpers";
 
-import { pushNotification, useTabStore } from "@/store";
-import {
-  SqlEditorActions,
-  SqlEditorState,
-  SheetGetters,
-  SqlDialect,
-} from "../../types";
+import { pushNotification, useTabStore, useSQLEditorStore } from "@/store";
+import { SheetGetters, SqlDialect } from "../../types";
```

Comment:
* `Sql` should be `SQL`;
* `"../../types"` could be `"@/types"`;

Reply:
Maybe rename the variable in a split PR, in here, only migrate the old one to pinia.